tests/test_generic_plotter.py

```diff
@@ -148,3 +148,33 @@ def test_screenshot(tmp_path):
     pl.plot(sphere)
     pl.show(screenshot=tmp_path / "screenshot.png")
     assert Path(tmp_path / "screenshot.png").is_file()
+
+
+def test_dark_mode():
```

Comment:
We are just checking whether they "run" or not, right? Are we doing some active checking of the results with dark mode? Maybe it would be worth it

Reply:
We are generating the screenshots for the image cache, the `verify_image_cache` fixture is in autouse.

These are the generated cache images for this test, we will be comparing against these images:

![dark_mode_3](https://github.com/user-attachments/assets/acde8b69-c90f-4103-8198-19f3ceb24a47)
![dark_mode](https://github.com/user-attachments/assets/fc58e35d-bb46-404d-a7ba-0cd97b80530a)
![dark_mode_1](https://github.com/user-attachments/assets/4936fb9d-d0c1-4bc4-9982-31bbea610027)
![dark_mode_2](https://github.com/user-attachments/assets/602957e1-7abc-4a4f-9f70-9fbb22b8df51)

Reply:
Aaah - I didn't realize it was on autouse. All good then!